src/cs50/sql.py

```diff
@@ -78,13 +50,25 @@ def connect(dbapi_connection, connection_record):
         # Register listener
         sqlalchemy.event.listen(self._engine, "connect", connect)
 
-        # Log statements to standard error
-        logging.basicConfig(level=logging.DEBUG)
+        # Configure logger
+        self._logger = logging.getLogger(__name__)
+        self._logger.setLevel(logging.DEBUG)
+
+        # Log messages once
+        self._logger.propagate = False
+
+        ch = logging.StreamHandler()
```

Comment:
What's `ch` connote?

Reply:
Copied from the docs. I think it's short for console handler? I'll rename to just handler.